examples/add_custom_searchbox.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+<meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
+<style>
+  html, body,#map {
+      width:100%; 
+      height:100%; 
+      padding: 0; 
+      margin: 0;
+      
+    }
+  div#searchbox{
+    background-color: #d2eaef;
+    opacity: 0.8;
+    position: absolute;
+    top: 10px;
+    left: 50px;
+    width: auto;
+    height: auto;
+    padding: 10px;
+    display: block;
+    z-index: 9000;
+
+    }
+    div#searchbox input{
+      width: 200px;
+    }
+ </style>
+   <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
+
+  </head>
+<body>
+  <div id="map"></div>
+  <div id="searchbox">
+  <input type="text" name="ad" value="" id="ad" size="10" />
+  <button type="button" id="searchButton">Search</button>
+
+<script src="http://libs.cartocdn.com/cartodb.js/v3/3.15/cartodb.js"></script>
+<script>
+  var input;
+  function main () {
+    // define map settings
+    var map = L.map('map', {
+      zoomControl: false,
+      center: [41.390205, 2.154007],
+      zoom: 4
+    });
+    // add basemap to our map object
+    L.tileLayer('http://{s}.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png', {attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'}).addTo(map);
+
+    // add CartoDB layer
+    cartodb.createLayer(map, 'https://documentation.carto.com/api/v2/viz/34dfd2e4-d62b-11e5-b5e7-0ea31932ec1d/viz.json').addTo(map)
```

Comment:
I guess we could use `cartodb.createVis` here @oriolbx to make the example a bit easier:

```

cartodb.createVis(...)
.done(function(vis, layers) {
  ...
  var map = vis.getNativeMap();
  ...
  map.fitBounds(bounds);
  ...
});
```